### PR TITLE
chore: bump dependencies to fix vulnerability

### PR DESCRIPTION
**Pull request by Justintime50**
# Description

Fixes https://github.com/EasyPost/easypost-node/security/dependabot/52

# Testing

# Pull Request Type

Please select the option(s) that are relevant to this PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement (fixing a typo, updating readme, renaming a variable name, etc)

### CODE REVIEW

**Comment by nwithan8** [comment]
No changes committed to `packages.json`, just the lock file?

File: package-lock.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "@easypost/api",
-      "version": "6.7.3",
+      "version": "6.8.1",
```
Comment (jchen293) [inline package-lock.json:9]: Was this branch rebased correctly?

Comment (Justintime50) [reply package-lock.json:9]: Yes, why?

**Comment by Justintime50** [comment]
> No changes committed to `packages.json`, just the lock file?

Correct, the dependency with a vulnerability is a dep of a dep. We can't manipulate anything in `package.json` to fix that. A quick `npm update` bumps everything to the most recent versions that respect our pinned ranges, simultaneously bumping the vulnerable dep to what is required to use the fix.

@jchen293

**Review by nwithan8 — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (jchen293) [reply package-lock.json:9]: Ooo, looks we forgot to bump the version in package-lock.json when releasing the `6.8.1`

**Review by jchen293 — APPROVED** [review]

**Pull request merged**